src/standartizeUrl.js

```diff
@@ -0,0 +1,11 @@
+const standartizeURL = (url) => {
+  const textToConvert = url.split('?');
+  const params = new URLSearchParams(textToConvert[1]);
+  const toCase = params.get('toCase');
+
+  return [textToConvert[0].slice(1), toCase];
```

Comment:
Make it like a object, it would be better
```suggestion
  return { ...: textToConvert[0].slice(1), ...: toCase };
```

Reply:
i can't do this because convertToCase function accept 2 arguments instead of object